src/website/app/demos/layout/components/responsiveInstructions.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+const instructions = `
+1. Provide an array of \`breakpoints\`, which will be used in the (min-width)
```

Comment:
is min-width sufficient or do we need a max-width also?

Reply:
I'm hoping that min-width (aka a mobile-first approach) is enough. It's possible, though not necessarily straightforward to allow either/or in the future.